app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -25,7 +25,7 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
       }
 
       if (analysisUnavailable(analysis)) {
-        return "design";
+        return "";
```

Comment:
I sort of wonder if this `return ""` and in `PageRequestReview` should have a quick comment above them noting it's the Summary page like you did in `editCommonRedirects`, like "redirect to Summary page (slug root)". Not a must, but I'm not sure I'd know what was going on here.